template/src/store/reducers/index.js

```diff
@@ -8,7 +8,7 @@ import {MMKVStorage} from '../../services'
 export * from './app'
 export * from './user'
 
-export const InitialState = {
```

Comment:
please check this export

Reply:
if using `export` 
<img width="564" alt="Screen Shot 2023-01-06 at 14 08 43" src="https://user-images.githubusercontent.com/104345392/210949013-4c10db28-cf77-4c60-8d09-abbf38327bae.png">